modules/infra/submodules/storage/dominocost.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_s3_bucket" "costs" {
```

Comment:
Since we have all the other s3 buckets centralized in s3.tf, I would think this should be there too. I don't think there's anything that makes it so this should be separated out vs the others.